frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -42,7 +42,7 @@ import sbp from '@sbp/sbp'
 import { mapState, mapMutations } from 'vuex'
 import safeLinkTag from '@views/utils/safeLinkTag.js'
 import { CAPTURED_LOGS, SET_APP_LOGS_FILTER } from '@utils/events.js'
-import { downloadLogs, getLog } from '@model/captureLogs.js'
+import '@model/captureLogs.js'
```

Comment:
@snowteamer think this should go into `main.js` since `AppLogs.vue` is loaded lazyily... Even if it works right now, it might stop working if esbuild changes its behavior. Having it in `main.js` should ensure main.js's usage of it won't break.